packages/remix/package.json

```
@@ -99,7 +99,10 @@
     "test:integration:legacy": "export USE_OTEL=0 && run-s test:integration:v1 test:integration:v2",
     "test:integration:v1": "run-s test:integration:clean test:integration:prepare test:integration:client test:integration:server",
     "test:integration:v2": "export REMIX_VERSION=2 && run-s test:integration:v1",
-    "test:integration:ci": "run-s test:integration:clean test:integration:prepare test:integration:client:ci test:integration:server",
+    "test:integration:ci": "run-s test:integration:ci:otel test:integration:ci:legacy",
+    "test:integration:ci:otel": "export USE_OTEL=1 && run-s test:integration:ci:common",
```

```suggestion
    "test:integration:ci:otel": "export USE_OTEL=1 && yarn test:integration:ci:common",
```

---

or can we just make this:
```suggestion
    "test:integration:ci:otel": "USE_OTEL=1 test:integration:ci:common",
```